packages/data-stores/src/queries/use-launchpad.ts

```diff
@@ -69,14 +70,27 @@ export const fetchLaunchpad = (
 		  } as APIFetchOptions );
 };
 
+const mapTask = ( task: Task, index: number ) => {
```

Comment:
Nit: it may be better to call this something more self-descriptive, like `addOrderToTask`.
```suggestion
const addOrderToTask = ( task: Task, index: number ) => {
```